beachball.config.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path');
+const fs = require('fs');
+const os = require('os');
+
```

Comment:
Since this is duplicated from Nimble it could be worth adding a comment to both copies that links them to each other to remind us to keep them in sync if one of them changes.

Reply:
Maybe we should actually publish this as a package... I can see it being useful for other projects. For example I'd like to integrate it in Vireo as well.

Reply:
Even better!

Reply:
That package is in-work, I'll close this out for now and expect to integrate that package when ready: https://github.com/ni/nimble/pull/248